aes-kw/src/error.rs

```diff
@@ -28,7 +28,7 @@ pub enum Error {
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Error::InvalidDataSize => write!(f, "data must be a multiple of 64 bits"),
+            Error::InvalidDataSize => write!(f, "data must be a multiple of 64 bits for AES-KW and less than 2^32 bytes for AES-KWP"),
```

Comment:
Perhaps this error should be split up for the two different algorithms? On the other hand, I didn't want to break backwards compatibility.

Reply:
Kind of a verbose error message. Wish the `Error` enum were marked `non_exhaustive`.

Reply:
Is there a backwards-compatible way to solve this? Adding new variants might break pattern matching?

Reply:
No, but really, making a breaking release isn't that much of a problem, and if we mark the `Error` enum as `#[non_exhaustive]` it will prevent future breaking changes by adding error variants.